registry/tasks.go

```diff
@@ -83,10 +84,15 @@ func PurgeOldTags(client *Client, purgeDryRun bool, purgeTagsKeepDays, purgeTags
 		sort.Sort(sortedTags)
 		repos[repo] = sortedTags
 
-		// Filter out tags by retention days.
+		// Filter out tags by retention days and regexp
 		for _, tag := range repos[repo] {
+			regexpKeep := false
+			if purgeTagsKeepRegexp != "" {
+				regexpMatch, _ := regexp.MatchString(purgeTagsKeepRegexp, tag.name)
+				regexpKeep = regexpMatch
```

Comment:
```
				regexpMatch, _ := regexp.MatchString(purgeTagsKeepRegexp, tag.name)
				regexpKeep = regexpMatch
```
can be simplified to
```
				regexpKeep, _ = regexp.MatchString(purgeTagsKeepRegexp, tag.name)
```

Reply:
Not much go experience, wasn't sure that that worked with `=` too or that `something, _` always had to be `:=`